bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gmos/GmosNorthType.java

```diff
@@ -80,27 +80,29 @@ public StageModeNorth parse(String name, StageModeNorth defaultValue) {
         }
     };
 
-    public static enum DisperserNorth implements GmosCommonType.Disperser, ObsoletableSpType {
-        MIRROR("Mirror", "mirror"),
-        B1200_G5301("B1200_G5301", "B1200"),
-        R831_G5302("R831_G5302", "R831"),
-        B600_G5303("B600_G5303", "B600") {public boolean isObsolete() {return true;}},
-        B600_G5307("B600_G5307", "B600"),
-        R600_G5304("R600_G5304", "R600"),
-        R400_G5305("R400_G5305", "R400"),
-        R150_G5306("R150_G5306", "R150"),
-        R150_G5308("R150_G5308", "R150"),
+    public enum DisperserNorth implements GmosCommonType.Disperser, ObsoletableSpType {
+        MIRROR("Mirror", "mirror", 0),
```

Comment:
You could argue that the lines per mm value should be an optional since the mirror doesn't have it. However I think the actual issue here is that the mirror isn't a disperser/grating and if anything should be an optional value then it is the disperser element. *Every* disperser element must have a "lines per mm" value. So I was a bit reluctant to model this as an optional in order to deal with a design flaw(?) somewhere else.